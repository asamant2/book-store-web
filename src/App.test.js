import {render} from '@testing-library/react';
import App from './App';

describe('App Component', () => {
    it("should render App with error", () => {
        const { asFragment } = render(<App/>);
        expect(asFragment()).toMatchSnapshot();
    })
})