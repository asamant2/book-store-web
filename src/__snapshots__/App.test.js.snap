// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render App with error 1`] = `
<DocumentFragment>
  <h1>
    My first component.
  </h1>
</DocumentFragment>
`;
